### ISSUE TITLE
bug: error returned from database: permission denied for table worker_group_config

### ISSUE THREAD

**Issue by pka** [windmill-labs/windmill#2281]
### Describe the bug

After updating to 1.172.0 the following error appears in the application log when opening the Workers page:

```
windmill-windmill-1              | 2023-09-14T06:08:22.999016Z ERROR request: windmill_common::error: error="Sql error: error returned from database: permission denied for table worker_group_config" method=GET uri=/api/workers/list_worker_groups username="***" email="***"
```
Database setup has been done according to https://github.com/windmill-labs/windmill#postgres-without-superuser

It *might* have been caused by an error during migration. The disk was full during updating.

### To reproduce

1. Go to 'https://windmill.example.com/workers'
2. See error in log

### Expected behavior

No error in log. Worker page itself looks ok.

### Application version

ghcr.io/windmill-labs/windmill:1.172.0

**Comment by rubenfiszel** [comment]
I know what this is, I will push a fix soon

**Comment by pka** [comment]
No hurry!

**Comment by rubenfiszel** [comment]
should be fixed in 1.172.1 (still building)

**Comment by pka** [comment]
Errror message is gone, thanks!
